ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -204,7 +204,7 @@ class SourcePointClient: SourcePointProtocol {
             "messageId": messageId,
             "includeData": "{\"categories\": {\"type\": \"RecordString\"}}"
         ])! // swiftlint:disable:this force_unwrapping
-        client.get(urlString: url.absoluteString) { result in
+        client.get(urlString: url.absoluteString, apiCode: nil) { result in
```

Comment:
@Nevazhnovu we can add apiCodes for these endpoints with `apiCode: nil`, feel free to pick a name you think is most descriptive and I'll review them.

Reply:
@andresilveirah done